Libraries/LibJS/Runtime/StringPrototype.cpp

```diff
@@ -185,6 +186,40 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::starts_with)
     return Value(string.substring(start, search_string_length) == search_string);
 }
 
+JS_DEFINE_NATIVE_FUNCTION(StringPrototype::ends_with)
+{
+    auto string = ak_string_from(vm, global_object);
+    if (string.is_null())
+        return {};
+    if (!vm.argument_count())
+        return Value(false);
+
+    auto search_string = vm.argument(0).to_string(global_object);
+    if (vm.exception())
+        return {};
+
+    auto string_length = string.length();
+    auto search_string_length = search_string.length();
+
+    size_t end = string_length;
+    if (vm.argument_count() > 1) {
+        auto number = vm.argument(1).to_number(global_object);
+        if (vm.exception())
+            return {};
+        end = min(string_length, number.to_size_t(global_object));
```

Comment:
Please do not use to_size_t anymore, it's not part of the spec - use the function that's actually being used instead :)